README.md

```diff
@@ -1,4 +1,55 @@
+![puta](https://github.com/PactCoffee/js-ui-challenge/raw/master/puta.png)
+
 Pact's front end challenge
 ==========================
 
-TBA
+The general idea
+----------------
+
+We have a sign up flow affectionately known as "The Funnel". It's primary use is to sign up customers to our service. They enter their information, it gets sent to the server. **The challenge is to build a very simple version of [our current funnel](http://pactcoffee.com/funnel/intro)**. Sounds easy enough, right?
+
+
+But wait! There's more...
+-------------------------
+
+1. The funnel is separated into "stages". Each stage collects certain information from the user. For this challenge, those stages are:
+  - Name & email address
+  - Brew type selection
+  - Delivery address
+  - Confirmation of everything
+
+1. The stages fill the viewport, and are laid out in a vertical-scrolling list
+
+1. Data from each stage is saved incrementally. If a user leaves the funnel and comes back, their data should still be there
+
+1. We may want to do multivariate testing of the funnel in future. As such, the architecture of the funnel needs to be flexible enough to allow for the following:
+
+  - The order of the stages being changed
+  - Stages being added or removed
+  - The flow between stages being changed (e.g. horizontal scroll instead of vertical)
```

Comment:
horizontal / vertical feels like a leading hint towards single page architecture where as the current funnel you've linked as an example is not. Is that intended?